cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,26 @@
+
+import { homePage } from "../support/pages/homePage"
+import testData from "../fixtures/testData.json"
+
+describe('Amazon Home page tests', () => {
+
+  beforeEach ('open home page', () => {
+    cy.visit('/')
```

Comment:
put it to the home page like openHopePage(){...}

Reply:
fixed